From 6df81146105b90f6b97527994ef1d00a1fa9dc93 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:11:36 -0400
Subject: [ethernet] i40evf: remove debugging message

Message-id: <1409663591-32531-54-git-send-email-sassmann@redhat.com>
Patchwork-id: 90527
O-Subject: [RHEL7.1 PATCH 053/148] i40evf: remove debugging message
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

If the PF driver fails or is removed from the host, the VF driver will
fill up its log with this message.

Change-ID: I67045f987f7c0d444d21ded403adc509343cdb8f
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Sibai Li <sibai.li@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit c50d2e5d94e1405d739a0656c3a1943688dd629e)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index d7abd0d..0a4ff41 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -1311,7 +1311,6 @@ static void i40evf_watchdog_task(struct work_struct *work)
   goto restart_watchdog;
 
  if (adapter->flags & I40EVF_FLAG_PF_COMMS_FAILED) {
-  dev_info(&adapter->pdev->dev, "Checking for redemption\n");
   if ((rd32(hw, I40E_VFGEN_RSTAT) & 0x3) == I40E_VFR_VFACTIVE) {
    /* A chance for redemption! */
    dev_err(&adapter->pdev->dev, "Hardware came out of reset. Attempting reinit.\n");
-- 
1.7.1